introducing-pgstream.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: 'Introducing pgstream: Postgres replication with DDL changes'
+description: 'Today we’re excited to expand our open source Postgres platform with pgstream, a CDC command line tool and library for PostgreSQL with replication support for DDL changes to any provided output.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pgstream/introducing-pgstream@2x.jpg
+  alt: pgstream
+author: Esther Minano Sanz
+authorEmail: esther@xata.io
+date: 07-17-2024
+tags:
+  [
+    'open-source',
+    'postgres',
+    'fpReplication',
+    'fpPgstream',
+    'fpPgroll',
+    'fpPgzx',
+    'oss',
+    'launch-week',
+    'elephant-on-the-move'
+  ]
```

Comment:
@CezzaineZ or @alexfrancoeur, can you confirm which feature pages this blog post should be included in? The tags starting with `fp` add the blog post to the `Explore more resources` section of a specific feature page.

Reply:
Oh I added `fpReplication` but if it's a feature page, it will not exist. It probably should be just `replication` as a tag in that case.